docs/docs/api/Strapi.mdx

```diff
@@ -0,0 +1,525 @@
+
+```javascript
+const strapiInstance = new Strapi();
+```
+
+### `strapi.container`
+- [<Type>Container</Type>](./container)
+
+This is where all registries are stored.
+
+See [Container](./container).
+
+### `strapi.dirs`
+- <Type>Object</Type>
+
+Stored paths of file system.
+
+- `dirs.dist`: <Type>[StrapiPathObject](#strapipathobject)</Type>
+  - Build folder
+- `dirs.app`: <Type>[StrapiPathObject](#strapipathobject)</Type>
+  - Sources folder
+- `dirs.static`: <Type>Object</Type> Define path to directories involving web client display
+  - `public`:  <Type>String</Type> Path to the folder to serve publicly (like files, images, etc..)
+
+#### StrapiPathObject
+- <Type>Object</Type>
+
+A set of path to specific Strapi project parts.
+
+- `root`: <Type>String</Type> Root path
+- `src`: <Type>String</Type> Sources route path to project files
+- `api`:  <Type>String</Type> Path to the folder containing project developers' API files (content-types, controllers, services, routes, etc..)
+- `components`:  <Type>String</Type> Path to the folder containing project developers' components
+- `policies`:  <Type>String</Type> Path to the folder where the Strapi project developers' policies are stored
+  - A set of functions that check the state of the data and prevent the access to the API accordingly
+- `middlewares`:  <Type>String</Type> Path to the folder where the Strapi project developers' middlewares are stored
+  - A set of function that wrap around routes and requests
+- `config`:  <Type>String</Type> Path to the folder containing project developers' config files
+
+### `strapi.isLoaded`
+- <Type>Boolean</Type>
+
+  - `true`: Everything have been loaded
+  - `false`: There is something loading
+
+### `strapi.reload()`
+
+Reload the app.
+
+This function defines itself at the construction of the Strapi class.
+
+### `strapi.server`
+- [<Type>StrapiServer</Type>](./strapi-server)
+
+Strapi server object.
+
+### `strapi.fs`
+- [<Type>StrapiFS</Type>](StrapiFS)
+
+Wrapper around [FS NodeJS module](https://nodejs.org/docs/latest-v18.x/api/fs.html).
+
+### `strapi.eventHub`
+- [<Type>EventHub</Type>](EventHub)
+
+The `strapi.eventHub` object is used to manipulate events within a Strapi project. It is an instance of the built-in EventEmitter class from Node.js, which provides a simple way to emit and listen for events.
+
+The `strapi.eventHub` object is created using the `createEventHub()` function in the [EventHub](EventHub) module of the Strapi core. This function returns a new instance of the EventHub class, which extends the EventEmitter class and adds some additional functionality specific to Strapi.
+
+
+#### Examples:
+
+```javascript
+// Listen for a 'user.updated' event and log the data
+strapi.eventHub.on('user.updated', (data) => {
+console.log(`User ${data.id} has been updated`);
+});
+
+// Emit a 'user.created' event with some data
+strapi.eventHub.emit('user.created', { username: 'johndoe', email: 'johndoe@example.com' });
```

Comment:
this is so cool :) 

@remidej  Do we have docs on the current events we are sending? I think it could be super useful to track them in a single document here. wdyt?

Reply:
We don't currently have that, but it's an interesting idea indeed!